.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+  - 2.12.9
+
+script:
+  - sbt clean compile test it:test
```

Comment:
We've already added alias for `test it:test`. 
For convenience you could use it: `sbt clean compile testAll`

Reply:
Also before we merge this PR we need to return one dependency after previous PR.
@likeanowl could we return `testContainers `?

Thank you in advance!